tests/phpunit/tests/html-api/wpHtmlProcessor.php

```diff
@@ -227,9 +280,9 @@ public function data_void_tags() {
 	 * @param string $tag_name Name of the tag to test.
 	 */
 	public function test_step_in_body_fails_on_unsupported_tags( $tag_name ) {
-		$fragment = WP_HTML_Processor::create_fragment( '<' . $tag_name . '></' . $tag_name . '>' );
-		$this->assertFalse( $fragment->next_tag(), 'Should fail to find tag: ' . $tag_name . '.' );
-		$this->assertEquals( $fragment->get_last_error(), WP_HTML_Processor::ERROR_UNSUPPORTED, 'Should have unsupported last error.' );
+		$p = WP_HTML_Processor::create_fragment( '<' . $tag_name . '></' . $tag_name . '>' );
+		$this->assertFalse( $p->next_tag(), 'Should fail to find tag: ' . $tag_name . '.' );
+		$this->assertEquals( $p->get_last_error(), WP_HTML_Processor::ERROR_UNSUPPORTED, 'Should have unsupported last error.' );
```

Comment:
for new tests I've tried to standardize on `$processor` after receiving feedback that `$p` was unclear for some, confusing it with a `P` element. would be good to do a full rename throughout the tests to unify the examples.

Reply:
That seems sensible, I reckon it can go against the generic test improvement tickets for 6.5 WP#59647 on trac.

Reply:
Thanks, I'll remove this change and put together a PR with those renames.

Reply:
PR for renames: https://github.com/WordPress/wordpress-develop/pull/5987